src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java

```diff
@@ -188,4 +188,26 @@ private Pkg() { }
         return new LocalCall<>("pkg.info_available", Optional.of(Arrays.asList(packages)),
                 Optional.empty(), new TypeToken<Map<String, Info>>(){});
     }
+
+    public static LocalCall<Boolean> upgradeAvailable(String packageName) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
+        args.put("package", packageName);
+        return new LocalCall<>("pkg.upgrade_available", Optional.empty(), Optional.of(args),
+                new TypeToken<Boolean>(){});
+    }
+
+    public static LocalCall<String> latestVersion(String packageName) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
+        args.put("package", packageName);
+        return new LocalCall<>("pkg.latest_version", Optional.empty(), Optional.of(args),
+                new TypeToken<String>(){});
+    }
+
+    public static LocalCall<Map<String, String>> latestVersions(String firstPackageName,
+            String secondPackageName, String...packages) {
+        return new LocalCall<>("pkg.latest_version",
+                    Optional.of(Arrays.asList(firstPackageName,
+                            secondPackageName, packages)),
+                Optional.empty(), new TypeToken<Map<String, String>>(){});
+    }
```

Comment:
It is necessary to have those three parameters here in order to make sure that there is at least two package names were actually given, right? That would make sense in case Salt decides on the return type based on the incoming parameters. I would suggest to still use the same name as the name of the Salt module though (`latestVersion()`) just to make clear that it maps to the same module. Apart from that indentation looks a but weird here, maybe it could be improved a bit?